src/components/navbar/navbar.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import "./navbar.css";
+
+const Navbar = () => {
+  const [showNav, setShowNav] = useState(" ");
+  const hamburgerClick = () => {
+    if (showNav === " ") setShowNav("mobile-navbar-view");
+    if (showNav === "mobile-navbar-view") {
+      setShowNav(" ");
+    }
+  };
```

Comment:
A simple boolean state can be managed here instead of checking value

Reply:
Yes I will change it